tutorials/inputs/input_examples.rst

```diff
@@ -42,6 +42,27 @@ Examples:
             # move as long as the key/button is pressed
             position.x += speed * delta
 
+ .. code-tab:: csharp
+
+    // Input event - runs when the input happens.
+    public override void _Input(InputEvent inputEvent)
+    {
+        if (inputEvent.IsActionPressed("jump"))
+        {
+            Jump();
+        }
+    }
+
+    // Polling - runs every frame.
```

Comment:
Remove this `// Polling` line and remove the `// Input event` line. They are not necessary comments and do not really add any information.